tests/installation/yast2_firstboot.pm

```diff
@@ -64,7 +64,7 @@ sub clock_and_timezone {
 sub user_setup {
     my $is_not_shared_passwd = shift;
     assert_screen 'local_user';
-    enter_userinfo(username => get_var('YAST2_FIRSTBOOT_USERNAME'));
+    $self->enter_userinfo(username => get_var('YAST2_FIRSTBOOT_USERNAME'));
```

Comment:
Actually, I think it should be called without self.
Because the function doesn't need it nor use it.
I actually just added it because it was called with self and that caused wrong parameters.

Reply:
Because for the self to work, y2_installbase should use the esoteric multi parent class:
```perl
use parent qw(installation_user_settings y2_installbase);
```

Reply:
Yes, but you have added it in the recent PR.

Reply:
So what is your proposal? To change it back in `enter_userinfo`? Because in other calls, it's is called with self and this one was the only one remaining.

Reply:
I actually just added it because it was called with self and that caused wrong parameters.

Reply:
But now I see that it shouldn't be called with self

Reply:
WDYT about https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9833 ?

Reply:
Yep, let's go for that one.